scripts/gdal/tests/gdal_preset_test.py

```diff
@@ -43,6 +43,26 @@ def test_preset_lzw() -> None:
     assert "EPSG:2193" in gdal_command
 
 
+def test_preset_dem_lerc() -> None:
+    gdal_command = get_gdal_command("dem_lerc", epsg="2193")
+    # Basic cog creation
+    assert "COG" in gdal_command
+    assert "blocksize=512" in gdal_command
+    assert "num_threads=all_cpus" in gdal_command
+    assert "bigtiff=yes" in gdal_command
```

Comment:
I am wondering if we need `bigtiff=yes` do we ever plan on making tiffs bigger than 4GB? I would hope not. so maybe we should make this `no` and then GDAL will fail if we ever accidentally make a tiff too big ?

Reply:
@dwsilk, in a huddle we could not remember/find reason why we have `Bigtiff` always set to `yes`, do you have any thoughts/can recall?

Reply:
Not something we used in our original process. The raster files we work with are always under 4GB.

Reply:
Only circumstances I can think we'll need it is if we're working with datasets like adhoc satellite imagery procured for emergency response.

Reply:
So that we can better document the conversation and in order to complete this ticket, this conversation will be continued in [TDE-805](https://toitutewhenua.atlassian.net/browse/TDE-805).

Reply:
@MDavidson17 perfect, lets go ahead and merge this then!